.idea/.idea.SS14.Admin/.idea/.name

```diff
@@ -1 +0,0 @@
-SS14.Admin
```

Comment:
No Clue why or how this was deleted someone tell me if its important and ill add it back

Reply:
The solution to this is to add `.idea/` to the .gitignore

Reply:
Fixed, added .name back and to my .gitignore cheers